status-page.py

```diff
@@ -1,72 +1,133 @@
 import json
 import sys
+from urllib.parse import urlparse
+
 import requests
 
 
 class StatusPage:
     def __init__(self):
         config_data_file = open(sys.argv[1])
-        self.test_results = open(sys.argv[2])
+        log_file = open(sys.argv[2])
         self.config_json = json.load(config_data_file)
+        self.log_json = json.load(log_file)
         self.access_token = self.config_json['access_token']
         self.base_url = self.config_json['base_url']
         self.header = {'x-cachet-token': self.access_token}
-        self.components = {
-            'operational': [],
-            'performance_issues': [],
-            'partial_outage': [],
-            'major_outage': []
-        }
-        self.incidents = {
-            'investigating': [],
-            'identified': [],
-            'watching': [],
-            'fixed': []
-        }
-        self.passed_tests = []
-        self.failed_tests = []
-
-    def parseTestResults(self):
-        self.passed_tests = []
-        self.failed_tests = []
+        self.broken_components = []
+        self.open_incidents = []
 
-    def getAllStatuses(self):
+    def get_broken_components(self):
```

Comment:
Can you add doc strings to these functions? Example: https://github.com/osu-mist/express-api-skeleton/blob/master/src/tests/integration/utils.py#L39